web/src/components/njwds-extended/UnStyledButton.tsx

```diff
@@ -1,87 +1,46 @@
-import { CircularProgress } from "@mui/material";
 import React, { forwardRef, ReactElement, useEffect, useRef, useState } from "react";
+
 interface Props {
-  style: "tertiary" | "light" | "narrow-light" | "unstyled";
+  style: "default" | "transparentBgColor";
```

Comment:
I'm seeing a number of instances where we explicitly set `style="default"`. Could `default` be applied by default? That makes a little more sense given the naming.